Assets/images/footer/facebook_footer.svg

```diff
@@ -4,17 +4,17 @@
 <svg
    version="1.1"
    id="svg1"
-   width="448"
-   height="445"
-   viewBox="0 0 448 445"
+   width="50.000031"
```

Comment:
Changing values here has no effect on file size

Reply:
The comment is relevant only because it is related to a task about file size.
If we look at the PR description all is good